### ISSUE TITLE
Analytics Debugger not working "No tools were detected"

### ISSUE THREAD

**Issue by rich13579** [analytics-debugger/analytics-debugger-browser-extension#23]
**Describe the bug**
Hi, after I press 'Start Debugging', I am shown a spinning wheel with a message 'No tools were detected, currently enabled vendors/tools...'
If I click on 'Settings', I see a list of tools and areas where I am apparently supposed to drag them to enable them.  However I am unable to drag them.
Hence I am unable to commence debugging of any sort.

**Expected behavior**

**Screenshots**
See attached 
<img width="1224" alt="Screenshot 2023-01-14 at 20 39 29" src="https://user-images.githubusercontent.com/86974118/212490656-91bb01b6-853c-4595-b373-c6b5b26dd7bc.png">
<img width="1588" alt="Screenshot 2023-01-14 at 20 39 36" src="https://user-images.githubusercontent.com/86974118/212490658-09035d41-2f96-4c49-9fe5-0a8615f25ce3.png">

**Environment (please complete the following information):**
macOS Ventura 13.0.1
Chrome Version 109.0.5414.87 (Official Build) (x86_64)
 - Version [e.g. 105.xx.xxx]

**Additional context**

Best,

Richard

**Comment by DEVJhonCortes** [comment]
**Same error here!**

In MacOS and Windows.

Browser:
- Chrome (Version 109.0.5414.87 (Build oficial) (arm64)
- Brave (Version 1.47.171 Chromium: 109.0.5414.87 (Build oficial) (arm64)

**Comment by thyngster** [comment]
I'm not able to replicate this anywhere ...  which is really weird, have you guys tried to reinstall or disable other extensions?

**Comment by DEVJhonCortes** [comment]
Hi David,

The extension had been working fine until yesterday when it stopped working for me. It keeps loading and does nothing.

I have tried on another computer, in several browsers, reinstall the extension, clear cache, clear cookies; I did everything but the error still persists.

On a Windows computer i got the same issue reported by rich13579 and on macOS 13.1 it does nothing. It is as if there was a problem loading some library, since it does not load the news section either.

<img width="697" alt="image" src="https://user-images.githubusercontent.com/52833084/213018911-87254e95-1195-4f98-b509-22a0d17bc045.png">

<img width="690" alt="image" src="https://user-images.githubusercontent.com/52833084/213018992-08f227f0-90f2-4ee0-85e7-65c8ac840e36.png">

When I try to check the debug it keeps loading and then it looks like this:

<img width="306" alt="image" src="https://user-images.githubusercontent.com/52833084/213019309-651e7af1-e4cc-4756-aef1-2ee09dddc9fd.png">

Thanks

**Comment by thyngster** [comment]
I'm trying and it doesn't fail to me. Looking at your screenshots it seems the feed fetch fails and that may be causing the extension to load. it may be an issue with some firewall, or your connection. Is there any way for you to try with another connection?. 

In any case these seems to be different issues.

**Comment by thyngster** [comment]
I'm checking and if the feed endpoint is down, it won't cause any issues, looks more like your connection get stuck on connecting to me. So maybe there's some issues with my server connection for some ISPs. I'll be reviewing it

**Comment by DEVJhonCortes** [comment]
Hello David.

**I have checked the connection as mentioned and that is the problem.**

My ISP is ETB (Colombia) and if I try to access your website: https://www.thyngster.com/, it generates a timeout error.

I have temporarily switched to a network with ISP Claro and i have been able to access the website, and i tried the GTM/GA Debugger tool and it also worked. Therefore, the problem is that due to these connection problems with your website, the extension stops working.

I have also tried some Gtmetrix servers and in the case of Canada it generates the same error: https://gtmetrix.com/?job_error=hKCWyXK1

We are super attentive that the connection to your site can be solved or that the libraries can be accessible with a CDN (I don't know if you have it) so that these problems do not occur.

Thanks a lot, David.
